Calculator/ViewController.swift

```diff
@@ -8,50 +8,55 @@
 import UIKit
 
 class ViewController: UIViewController {
-
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
-        resultado()
+
+        resultLabel.text = defaults.string(forKey: "resultado")
+
+        let hide = defaults.bool(forKey: "enabled_preference")
+        hideButton(hide: hide)
     }
-    @IBOutlet weak var resultLabel: UILabel!
-    //Numbers
-    @IBOutlet weak var btn1: UIButton!
-    @IBOutlet weak var btn2: UIButton!
-    @IBOutlet weak var btn3: UIButton!
-    @IBOutlet weak var btn4: UIButton!
-    @IBOutlet weak var btn5: UIButton!
-    @IBOutlet weak var btn6: UIButton!
-    @IBOutlet weak var btn7: UIButton!
-    @IBOutlet weak var btn8: UIButton!
-    @IBOutlet weak var btn9: UIButton!
-    @IBOutlet weak var btn0: UIButton!
-    @IBOutlet weak var btnDecimal: UIButton!
-    
-    //Operators
-    
-    @IBOutlet weak var btnRestart: UIButton!
-    @IBOutlet weak var btnDiv: UIButton!
-    @IBOutlet weak var btnMult: UIButton!
-    @IBOutlet weak var btnRes: UIButton!
-    @IBOutlet weak var btnSum: UIButton!
-    @IBOutlet weak var btnEqual: UIButton!
-    
+
+    let defaults = UserDefaults.standard
+    @IBOutlet var resultLabel: UILabel!
+    // Numbers
+    @IBOutlet var btn1: UIButton!
+    @IBOutlet var btn2: UIButton!
+    @IBOutlet var btn3: UIButton!
+    @IBOutlet var btn4: UIButton!
+    @IBOutlet var btn5: UIButton!
+    @IBOutlet var btn6: UIButton!
+    @IBOutlet var btn7: UIButton!
+    @IBOutlet var btn8: UIButton!
+    @IBOutlet var btn9: UIButton!
+    @IBOutlet var btn0: UIButton!
+    @IBOutlet var btnDecimal: UIButton!
```

Comment:
The Outlet to decimal button is ok, cause you are doing something with it (hide it or not with the switch)